Assets/Mirror/Tests/WeaverTest.cs

```diff
@@ -5,7 +5,7 @@
 using UnityEngine;
 using UnityEditor.Compilation;
 
-using Mirror.Weaver;
+/*using Mirror.Weaver;
```

Comment:
Why are we disabling tests?   Major refactor like this really benefits from having working tests